doc/governance.rst

```diff
@@ -0,0 +1,124 @@
+
+.. _governance:
+
+===========================================
+Scikit-learn governance and decision-making
+===========================================
+
+The purpose of this document is to formalize the governance process used by the
+scikit-learn project, to clarify how decisions are made and how the various
+elements of our community interact.
+This document establishes a decision-making structure that takes into account
+feedback from all members of the community and strives to find consensus, while
+avoiding any deadlocks.
+
+This is a meritocratic, consensus-based community project. Anyone with an
+interest in the project can join the community, contribute to the project
+design and participate in the decision making process. This document describes
+how that participation takes place and how to set about earning merit within
+the project community.
+
+Roles And Responsibilities
+==========================
+
+Contributors
+------------
+Contributors are community members who contribute in concrete ways to the
+project. Anyone can become a contributor, and contributions can take many forms
+– not only code – as detailed in the contributors guide.
+
+Core developers
+---------------
+Core developers are community members who have shown that they are dedicated to
+the continued development of the project through ongoing engagement with the
+community. Being a core developer  allows contributors to more easily carry on
+with their project related activities by giving them direct access to the
+project’s repository and is represented as being an organization member on the
+scikit-learn GitHub organization. Core developers are expected to review code
+contributions,can merge approved pull requests,can cast votes for and against
```

Comment:
what do you mean by ``are expected to can``

Reply:
It's supposed to parse as "Core developers can cast votes" but I'm not a native speaker.